core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/upload/BitbucketcloudUploader.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2020-2023 The JReleaser authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jreleaser.model.internal.upload;
+
+import org.jreleaser.model.Active;
+import org.jreleaser.model.internal.common.Artifact;
+import org.jreleaser.mustache.TemplateContext;
+
+import java.util.Map;
+
+import static java.util.Collections.unmodifiableMap;
+import static org.jreleaser.mustache.Templates.resolveTemplate;
+
+/**
+ * @author Hasnae Rehioui
+ * @since 1.7.0
+ */
+public class BitbucketcloudUploader extends AbstractGitPackageUploader<org.jreleaser.model.api.upload.BitbucketcloudUploader, BitbucketcloudUploader> {
```

Comment:
This uploader should be registered with JsonSchemaGenerator in the same way as others uploaders have been registered.